pkg/universe.dagger.io/x/vgjm456@qq.com/helm/helm.cue

```diff
@@ -0,0 +1,93 @@
+package helm
+
+import (
+	"dagger.io/dagger"
+	// "dagger.io/dagger/core"
+
+	"universe.dagger.io/alpine"
+	"universe.dagger.io/bash"
+	// "universe.dagger.io/docker"
+)
+
+#Install: {
+	// Name of your release
+	name:       string | *""
+	kubeconfig: dagger.#Secret
+
+	source: *"repository" | "URL" | "local"
+
+	if source == "repository" {
+		chart:      string
+		repository: string
+	}
+
+	if source == "URL" {
+		URL: string
+	}
+
+	// -----
+
+	if source == "repository" {
+		chart:      string
+		repository: string
+		env: {
+			REPOSITORY: repository
+			CHART:      chart
+		}
+		script: contents: #"""
+				helm repo add somerepo $REPOSITORY
+				helm repo update
+				helm install \#(name) somerepo/$CHART \#(_generateName)
+				"""#
+	}
+
+	if source == "URL" {
+		URL: string
+		run: script: contents: #"""
+				helm install \#(name) \#(URL) \#(_generateName)
+				"""#
+	}
+
+	run: bash.#Run & {
+		input:  _base.output
+		always: true
+		mounts: "/root/.kube/config": {
+			dest:     "/root/.kube/config"
+			type:     "secret"
+			contents: kubeconfig
+		}
+	}
+
+	_base: #Build
+
+	_generateName: string | *""
+	if name == "" {
+		_generateName: "--generate-name"
+	}
+}
+
+#Build: {
+
+	helmVersion: string | *"v3.8.1"
+
+	kubectlVersion: string | *"v1.23.5"
+
+	bash.#Run & {
+		input: _base.output
+		script: contents: #"""
+                    curl -sSLO https://get.helm.sh/helm-\#(helmVersion)-linux-amd64.tar.gz
+                    tar -zxf helm-\#(helmVersion)-linux-amd64.tar.gz
+                    mv linux-amd64/helm /usr/local/bin/helm
+                    curl -sSLO https://dl.k8s.io/release/\#(kubectlVersion)/bin/linux/amd64/kubectl
+                    chmod +x kubectl
+                    mv kubectl /usr/local/bin/
+                    """#
+	}
+
+	_base: alpine.#Build & {
+		packages: {
+			bash: {}
+			curl: {}
+		}
+	}
```

Comment:
Put `_base` above `bash.#Run`, it's easier to read
Should be `bash: _` and `curl: _`